Tools/build/generate_opcode_h.py

```diff
@@ -233,4 +233,4 @@ def main(opcode_py, outfile='Include/opcode.h',
 
 
 if __name__ == '__main__':
-    main(sys.argv[1], sys.argv[2], sys.argv[3])
+    main(sys.argv[1], sys.argv[2], sys.argv[3], sys.argv[4])
```

Comment:
```suggestion
    main(*sys.argv[1:])
```
?

Reply:
Ugh, sorry, enabled auto-merge and then went into a meeting and didn't see this until after it landed.

This is a good suggestion, but given that almost nobody (AFAIK) runs `generate_opcode_h.py` by hand, it's probably not a super high priority one? So I may not file a separate PR just to do this. If the requirement to provide all arguments is bothering someone in practice sufficiently to motivate the change at some point, that's great.

Let me know if you disagree and you'd like a follow-up PR for this change.

Reply:
Yeah, not a big deal, just a pattern that I see periodically. Looks like in this case it's a fixed number of optional args anyway, rather than a list of files as I first guessed, so it makes less sense anyway.